service/server.py

```diff
@@ -0,0 +1,40 @@
+import grpc
+import hwsc_file_transaction_svc_pb2_grpc
+from concurrent import futures
+from enum import Enum
+from service.file_transaction_service import FileTransactionService
+
+
+class State(Enum):
+    """A current state class of file transaction service"""
+    AVAILABLE = 0
+    UNAVAILABLE = 1
+
+
+class Server:
+
+    def __init__(self):
+        self.__state = State.AVAILABLE
+
+    # TODO test
+    def get_state(self):
+        return self.__state
+
+    # TODO test
+    def set_state(self, new_state):
+        self.__state = new_state
+
+    # TODO test
+    def serve(self, port):
+        server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+        hwsc_file_transaction_svc_pb2_grpc.add_FileTransactionServiceServicer_to_server(FileTransactionService(self),
```

Comment:
`self` in `FileTransactionService(self)` is the same in line https://github.com/hwsc-org/hwsc-file-transaction-svc/pull/44/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR5

This is used to aggregate `Server` as part of the `FileTransactionService` to keep track of the `state`